src/neo-vm/ExecutionContext.cs

```diff
@@ -1,16 +1,13 @@
-﻿using System.Collections.Generic;
-using System.Runtime.CompilerServices;
+﻿using System.Runtime.CompilerServices;
 
 namespace Neo.VM
 {
     public class ExecutionContext
     {
-        private readonly Dictionary<int, Instruction> instructions = new Dictionary<int, Instruction>();
-
         /// <summary>
         /// Number of items to be returned
         /// </summary>
-        internal int RVCount { get; }
+        public int RVCount { get; }
```

Comment:
Why change this to `public`?

Reply:
Because is read only, and maybe we need it for debuggers or something like that